### PR TITLE
Add registration view - refs #78 / feature/36498

### PR DESCRIPTION
**Pull request by Januszpl**
Add registration view
Feature/36498
Refs: #78

### CODE REVIEW

File: docs/styles/_components.scss [hunk 1]
```diff
@@ -103,3 +103,4 @@
 @import '../../components/04-views/catalog/grid/catalog-grid';
 @import '../../components/04-views/dashboard/default/dashboard';
 @import '../../components/04-views/home/home';
+@import '../../components/04-views/registration/registraiton';
```
Comment (kruchy8) [inline docs/styles/_components.scss]: typo in scss filename 🤓

File: components/04-views/registration/registration.hbs [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{{ render '@header' }}
+
+<main class="container registration">
+
+    <h1 class="heading registration__title">
```
Comment (kruchy8) [inline components/04-views/registration/registration.hbs]: h1 and heading share the same code: 
(from components/01-globals/typography/heading/_heading.scss )
```
h1,
h2,
h3,
h4,
h5,
h6 {
    @extend .heading;
}
```
If so..is it necessary to use this `heading` class here ?

File: docs/styles/_components.scss (hunk 1 above)
Comment (Januszpl) [reply docs/styles/_components.scss]: thx done

File: components/04-views/registration/registration.hbs (hunk 2 above)
Comment (Januszpl) [reply components/04-views/registration/registration.hbs]: no. thx. part of html structure was copied from other project not to write from scratch and forgot to check if it's necessary.

File: components/04-views/registration/_registration.scss [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+        border-radius: $registration__password-strength-radius;
+        background-color: $registration__password-bg-color;
+        padding: $registration__password-strength-padding;
+        color: $registration__password-color;
+        &:before {
+            .registration & {
+                background-color: $registration__password-bg-color;
+            }
+        }
+        span {
```
Comment (kruchy8) [inline components/04-views/registration/_registration.scss]: 😱  mother of specificities, do we really have to use this selector here ?

Comment (Januszpl) [reply components/04-views/registration/_registration.scss]: sure not. thx. done.
ps. 😱  mother of professionalism, do we really add scary icons to code review comment?

**Pull request merged**